Code/Framework/AzFramework/AzFramework/TargetManagement/TargetManagementAPI.h

```diff
@@ -236,7 +236,7 @@ namespace AzFramework
         typedef AZ::EBus<TargetManager> Bus;
 
         static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single; // there's only one target manager right now
-        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;  // theres only one target manager right now
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single; // there's only one target manager right now
```

Comment:
You can make an `AZ::Interface<TargetManager>` alias for the target manager so that it can be accessed using a singleton pattern

Reply:
That's a good call. It may make sense for TargetManagement to handle multiple hosts in the future but that shouldn't cause any issues.

Reply:
After thinking about this more, I'm going to defer this for potential work around supporting multiple targets. It's a good idea for certain but combined with the potential API changes and the low footprint of this EBUS so far, handling this in a future PR makes sense to me.